File: client/src/components/editor/_trim_sidebar.py
from pathlib import Path

path = Path(__file__).with_name("EditorSidebar.jsx")
text = path.read_text(encoding="utf-8")
start = text.find("const __REMOVE_FRAMES_LEFTOVER__")
end = text.find("const TextPanel")
if start == -1 or end == -1 or end <= start:
    raise SystemExit(f"markers not found start={start} end={end}")
path.write_text(text[:start] + text[end:], encoding="utf-8")
print(f"removed {end - start} chars")

File: client/src/components/editor/_trim_sidebar.js
const fs = require('fs');
const path = require('path');
const file = path.join(__dirname, 'EditorSidebar.jsx');
const text = fs.readFileSync(file, 'utf8');
const start = text.indexOf('const __REMOVE_FRAMES_LEFTOVER__');
const end = text.indexOf('const TextPanel');
if (start === -1 || end === -1 || end <= start) {
  throw new Error(`markers not found start=${start} end=${end}`);
}
fs.writeFileSync(file, text.slice(0, start) + text.slice(end));
console.log('removed', end - start, 'chars');
